eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -980,6 +980,32 @@ public function testUpdateLocationStructValues(array $data)
      *
      * @see \eZ\Publish\API\Repository\LocationService::updateLocation()
      * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testUpdateLocationWithSameRemoteId()
+    {
+        $repository = $this->getRepository();
+
+        $locationId = $this->generateId('location', 5);
+        /* BEGIN: Use Case */
+        // $locationId and remote ID is the IDs of the same, existing location
+        $locationService = $repository->getLocationService();
+
+        $originalLocation = $locationService->loadLocation($locationId);
+
+        $updateStruct = $locationService->newLocationUpdateStruct();
+        // Remote ID of an existing location
+        $updateStruct->remoteId = 'f3e90596361e31d496d4026eb624c983';
+
+        // Throws exception, since remote ID is already taken
```

Comment:
is this comment maybe a left-behind? i think your goal is just don't throw the exception, isn't it?